src/pages/Homepage/LendByCategoryHomepage.vue

```diff
@@ -157,6 +157,47 @@
 				<h2 class="lender-quotes__header text-center small-12 columns">
 					What our lending <br class="so mo"> community thinks
 				</h2>
+				<div
+					class="lender-quotes__lender-quote--card small-10 medium-8 large-5 xlarge-4 columns"
+					v-for="lenderQuote in lenderQuotes"
+					:key="lenderQuote.name"
+				>
+					<kv-responsive-image
+						class="lender-quotes__lender-quote--lender-img"
+						:images="[
+							['small', lenderQuote.image],
+						]"
+						loading="lazy"
+						alt=""
+					/>
+					<div
+						v-html="lenderQuote.quote"
+						class="lender-quotes__lender-quote--quote"
+					>
+					</div>
+					<div
+						v-html="lenderQuote.attribution"
+						class="lender-quotes__lender-quote--attribution"
+					>
+					</div>
+					<div
+						v-html="lenderQuote.title"
+						class="lender-quotes__lender-quote--title"
+					>
+					</div>
+					<!-- TODO:
+					This is a little tricky, need to put more thought in here
```

Comment:
Oh interesting, you could add a modifier class with the loop index on the quote-card. And then in CSS use the index to position them.

Sortof like 
```
<div :class="quote-card quote-card--{index}">
  <div :class="quote-card__flourish">
  ...

.quote-card__flourish {
  position: absolute;
}

.quote-card--1 .quote-card__flourish {
  top: 1rem;
  left: 1rem;
  width: 3rem;
}
.quote-card--2 quote-card__flourish {
  top: 2rem;
  right: 0;
  width: 2.5rem;
}
```

Reply:
☝️ Like @ryan-ludwig  said. You could also target with `:nth-child(odd)` `:nth-child(even)` if you wanted to support an arbitrary number of quotes,